egs/iwslt18st/st1/local/parse_xml.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python2
```

Comment:
We don't recommend python2. It's OK for now, but I'll ask you to use python 3 later.